src/binary/Yast.cc

```diff
@@ -166,6 +168,80 @@ void set_ruby_source_location(VALUE file, VALUE lineno)
   YaST::ee.setLinenumber(FIX2INT(lineno));
 }
 
+/**
+ * Returns true if the function name in an UI user input function which returns
+ * a symbol.
+ * @param  function_name name of the function
+ * @return true/false
+ */
+static bool ui_input_function(const char *function_name)
+{
+    return strcmp(function_name, "UserInput") == 0  ||
+        strcmp(function_name, "TimeoutUserInput") == 0 ||
+        strcmp(function_name, "PollInput") == 0;
+}
+
+/**
+ * Returns true if the parameter is the :debugHotkey symbol.
```

Comment:
still I think it should be be exact symbol value in function name as I expect function provide some abstraction. maybe something like `Returns true if the input symbol starts debugging.`